lib/toolbox/bit_buffer.h

```diff
@@ -98,6 +98,8 @@ void bit_buffer_copy_bits(BitBuffer* buf, const uint8_t* data, size_t size_bits)
  * @param [in,out] buf pointer to a BitBuffer instance to copy into
  * @param [in] data pointer to the byte array to be copied
  * @param [in] size_bitss size of the data to be copied, in bits
+ * @note Parity bits are placed starting with the least significant bit of each byte and moving up.
```

Comment:
I think this is the "most" significant bit (https://en.wikipedia.org/wiki/Bit_numbering)

Reply:
Fixe, nice catch